src/features/smartSearch/components/filters/EmailHistory/DisplayEmailHistory.tsx

```diff
@@ -0,0 +1,77 @@
+import DisplayTimeFrame from '../DisplayTimeFrame';
+import { getTimeFrameWithConfig } from '../../utils';
+import messageIds from 'features/smartSearch/l10n/messageIds';
+import { Msg } from 'core/i18n';
+import UnderlinedMsg from '../../UnderlinedMsg';
+import UnderlinedText from '../../UnderlinedText';
+import useCampaigns from 'features/campaigns/hooks/useCampaigns';
+import useEmails from 'features/emails/hooks/useEmails';
+import { useNumericRouteParams } from 'core/hooks';
+import { EMAIL_SELECT_SCOPE, MESSAGE_KEY_BY_OP } from '.';
```

Comment:
I don't understand why you need an enum at all. Why not just use the three strings?